src/sdk/strategy.ts

```diff
@@ -0,0 +1,279 @@
+import { SecretKey } from '@nucypher/nucypher-core';
+import { ethers } from 'ethers';
+
+import { Alice } from '../characters/alice';
+import { Bob } from '../characters/bob';
+import { Enrico } from '../characters/enrico';
+import { tDecDecrypter } from '../characters/universal-bob';
+import { ConditionSet } from '../policies/conditions';
+import { EnactedPolicy } from '../policies/policy';
+import {
+  base64ToU8Receiver,
+  fromBase64,
+  toBase64,
+  u8ToBase64Replacer,
+} from '../utils';
+import { Web3Provider } from '../web3';
+
+import { Cohort, CohortJSON } from './cohort';
+
+type StrategyJSON = {
+  cohort: CohortJSON;
+  startDate: Date;
+  endDate: Date;
```

Comment:
This is an artifact from a "Vanilla PRE" Alice paying and publishing policy metadata to the `SubscriptionManager` contract.   Since the start and end dates here are not expressed as conditions, and nodes do not check the `SubscriptionManager` contract for payment of policy validity during CBD evaluation, these lines have no effect.

If these strategies are intended for use with PRE *and* CBD, let's just make it optional, if it's intended for CBD only, they can be removed entirely.

Reply:
at minimum, we need a blurb in the docs to inform developers these lines have no affect.